src/issue/screens/issue.screen.js

```diff
@@ -10,13 +10,14 @@ import {
   Platform,
 } from 'react-native';
 import { Icon } from 'react-native-elements';
-
+import moment from 'moment/min/moment-with-locales.min';
```

Comment:
Add a blank line after the moment import so we can separate external imports from local components.